Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,253 @@
+            'ID': issue.get('number'),
+            'Title': issue.get('title'),
+            'Body': issue.get('body'),
+            'Assignees': [assignee.get('login') for assignee in issue.get('assignees')],
+            'Labels': [label.get('name') for label in issue.get('labels')]
+        })
+    HR = tableToMarkdown("{} issues found.".format(res['total_count']), context,
+                         ["ID", "Title", "Body", "Assignees", "Labels"])
+    EC = {'GitHub.Issue(val.ID && val.ID == obj.ID)': context}
+    return_outputs(HR, EC, res)
+
+
+def fetch_incidents_command():
+    last_run = demisto.getLastRun()
+
+    last_fetch = last_run.get('time')
+    if last_fetch is None:
+        last_fetch = (datetime.now() - timedelta(days=1)).strftime('%Y-%m-%dT%H:%M:%SZ')
+    last_issue = datetime.strptime(last_fetch, '%Y-%m-%dT%H:%M:%SZ')
+
+    incs = []
+
+
+    url = "https://api.github.com/search/issues?q=repo:{}/{}+state:open+created:>{}".format(OWNER, REPO, last_fetch)
+    res = http_request("GET", full_url=url)
+    for issue in res.get('items'):
+        issue_time = issue.get('created_at')
+        issue_time_date = datetime.strptime(issue_time, '%Y-%m-%dT%H:%M:%SZ')
+        inc = {
+            'name': 'Issue number {}, titled: "{}"'.format(issue.get('number'), issue.get('title')),
+            'occured': issue_time,
+            'rawJSON': json.dumps(issue)
+        }
+        if (issue_time_date > last_issue):
+            last_issue = issue_time_date
+        incs.append(inc)
+    demisto.setLastRun({'time': last_issue.strftime('%Y-%m-%dT%H:%M:%SZ')})
+    demisto.incidents(incs)
+
+
+def main():
+    ## Global variables declaration
+    global REPO, OWNER, API, API_KEY
+    REPO = demisto.params().get('repo_name')
+    OWNER = demisto.params().get('repo_owner')
+    API_KEY = demisto.params().get('api_key')
+    API = 'https://api.github.com/repos/' + OWNER + "/" + REPO
+
+    '''EXECUTION CODE'''
+    COMMANDS = {
+        "list-all-issues": list_all_issues_command,
+        'create-issue': create_issue_command,
+        'close-issue': close_issue_command,
+        'update-issue': update_issue_command,
+        'download-count': download_count_command,
+        'search-issues': search_issues_command,
+        'fetch-incidents': fetch_incidents_command
+    }
+    command = demisto.command()
+    LOG('GitHub command is: %s' % (command,))
+    try:
+        if command == 'test-module':
+            headers = {'Authorization': 'Bearer ' + API_KEY}
+            r = requests.request("GET",
+                                 API,
+                                 headers=headers)
+            if (r.status_code == 200):
+                demisto.results('ok')
+            else:
+                demisto.results('Unable to connect with the given credentials.')
+            sys.exit(0)
+        cmd_func = COMMANDS.get(command)
+        if cmd_func is None:
+            raise NotImplemented('Command "%s" is not implemented.') % (cmd_func)
```

Comment:
`Command "None" is not implemented.`
```suggestion
            raise NotImplemented('Command "{}" is not implemented.'.format(command))
```